storybook/stories/Layout/PhoneNumberInput.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import { storiesOf } from "@storybook/react-native";
+import { PhoneNumberInput } from "../../../src/components/Layout/PhoneNumberInput";
+import { View } from "react-native";
+import { size } from "../../../src/common/styles";
+
+const PhoneNumberInputItem = (): JSX.Element => {
+  const [countryCodeValue, setCountryCodeValue] = useState("+65");
+  const [mobileNumberValue, setMobileNumberValue] = useState("");
+  return (
+    <View style={{ margin: size(3) }}>
```

Comment:
From seb's update, now `PhoneNumberInput` can have scan option as well. I think can add that one here

Reply:
PhoneNumberInput don't have the scan option.. `PhoneNumberInput` used by `ItemIdentifier`. so it act as scanner one

Reply:
Yea, not need for the scan option at this level